subgraphs/aave-governance/src/handlers.ts

```diff
@@ -84,6 +86,54 @@ export function getGovernance(): Governance {
   return governance;
 }
 
+export function createDelegateChange(
+  event: ethereum.Event,
+  toDelegate: string,
+  fromDelegate: string,
+  delegator: string
+): DelegateChange {
+  const delegateChangeId = `${event.block.timestamp.toI64()}-${event.logIndex}`;
+
+  const delegateChange = new DelegateChange(delegateChangeId);
+
+  delegateChange.delegate = toDelegate;
+  delegateChange.delegator = delegator;
+  delegateChange.previousDelegate = fromDelegate;
+  delegateChange.tokenAddress = event.address.toHexString();
+  delegateChange.txnHash = event.transaction.hash.toHexString();
+  delegateChange.blockNumber = event.block.number;
+  delegateChange.blockTimestamp = event.block.timestamp;
+  delegateChange.logIndex = event.logIndex;
+
+  return delegateChange;
+}
+
+export function createDelegateVotingPowerChange(
+  event: ethereum.Event,
```

Comment:
Is it possible to put these handlers in a shared directory for both OZ and aave?

Reply:
I'm not sure tbh. But there are some minor differences in the handler functions / schema that would make sharing a bit awkward.